packages/utils/src/test/database.ts

```diff
@@ -0,0 +1,111 @@
+import { TSMigrationGenerator } from "@mikro-orm/migrations"
```

Comment:
question(this-file): Have these DB utils been extracted from somewhere else, or are they new in this PR?

Reply:
This is the util-ized version of the same from product module. I will have a separate PR  coming up that will clean up the products repo to use the new utils.

Reply:
Cool, that was what I was chasing 😄